custom/icds_reports/ucr/data_sources/infrastructure_form.json

```diff
@@ -12,7 +12,8 @@
     "reach-dashboard-qa",
     "reach-test",
     "icds-cas",
-    "cas-lab"
+    "cas-lab",
+    "icds-cas-sandbox"
```

Comment:
this is probably unintentional addition

Reply:
This is from the branch he merged in https://github.com/dimagi/commcare-hq/pull/24006/commits/400f9acffe2bb9b25e3fcaa25cf57ba620806a85
That commit is part of master now, so shouldn't be an issue.